curriculum/challenges/english/25-front-end-development/workshop-greeting-card/673332a6b63f9f1e1f81ba9b.md

```diff
@@ -7,11 +7,13 @@ dashedName: step-16
 
 # --description--
 
-Now you can do something similar to add the emoji also after the title: create a `h1::after` selector, give it a `content` property and set the value to `" 🥳"` (note there is a space before the emoji).
+Now you can do something similar to add the emoji also after the title. 
+
+Create an `h1::after` selector, give it a `content` property and set the value to `" 🥳"` (note there is a space before the emoji).
```

Comment:
```suggestion
Create a selector that targets the pseudo-element `::after` of the `h1` element. Give it a `content` property and set its value to `" 🥳"` (note there is a space before the emoji).
```
Since they have already done the same thing in the previous step I would avoid to give them the selector